presto-main/src/test/java/io/prestosql/execution/TestSqlTaskExecution.java

```diff
@@ -580,7 +591,13 @@ public void testComplex(PipelineExecutionStrategy executionStrategy)
             }
 
             outputBufferConsumer.abort(); // complete the task by calling abort on it
-            TaskState taskState = taskStateMachine.getStateChange(TaskState.RUNNING).get(10, SECONDS);
```

Comment:
I think we can simplify it by:
```
TaskState taskState = taskStateMachine.getStateChange(taskStateMachine.getState()).get(10, SECONDS);
```
not that if `getState` returns `FINISHED`, future will complete immediately.